src/core/book-keeper.js

```diff
@@ -1,8 +1,32 @@
 'use strict';
 
+var events = require('./event/event');
+
 class BookKeeper {
-    constructor () {
+    constructor (eventBus) {
+        eventBus.emitEvent(events.FILL_REMINDER_QUEUE, {
```

Comment:
Emitting FILL_REMINDER_QUEUE was just for testing - the book keeper shouldn't actually do that on initialization